### PR TITLE
[M1183] Bleaching ingest

### PR DESCRIPTION
**Pull request by gridcell**

### CODE REVIEW

**Comment by gridcell** [comment]
[create bleaching ingestion templates/serializers/commands](https://trello.com/c/AIlnGlWn/1183-create-bleaching-ingestion-templates-serializers-commands)

File: src/api/ingest/benthicpit_serializer.py [hunk 1]
```diff
@@ -25,14 +25,15 @@ def benthic_attributes_choices():
...
-    observations_field = "data__obs_benthic_pits"
+    observations_fields = ["data__obs_benthic_pits"]
     ordering_field = "data__obs_benthic_pits__interval"
     additional_group_fields = CollectRecordCSVSerializer.additional_group_fields.copy()
     additional_group_fields.append("data__benthic_transect__label")
 
     header_map = CollectRecordCSVSerializer.header_map.copy()
     header_map.update(
         {
+            "Observation interval *": "data__obs_benthic_pits__interval",
```
Comment (saanobhaai) [inline src/api/ingest/benthicpit_serializer.py:36]: Was this really just overlooked previously? There's no similar field missing from fish belt?

File: src/api/ingest/utils.py [hunk 2]
```diff
@@ -4,18 +4,22 @@
...
+    BleachingCSVSerializer,
+    FishBeltCSVSerializer,
+)
 from api.models import (
     BENTHICLIT_PROTOCOL,
     BENTHICPIT_PROTOCOL,
     BLEACHINGQC_PROTOCOL,
     FISHBELT_PROTOCOL,
     HABITATCOMPLEXITY_PROTOCOL,
+    CollectRecord,
```
Comment (saanobhaai) [inline src/api/ingest/utils.py:18]: Why is this needed?

File: src/api/ingest/benthicpit_serializer.py (hunk 1 above)
Comment (gridcell) [reply src/api/ingest/benthicpit_serializer.py:36]: The observation interval was on the base serializer and I moved it out.  Yeah I think it was overlooked

File: src/api/ingest/utils.py (hunk 2 above)
Comment (gridcell) [reply src/api/ingest/utils.py:18]: Formatting change via `isort`
